ide_extensions.md

```diff
@@ -0,0 +1,161 @@
+# IDE Extensions
+## Overview
+An oft-requested feature is an improved `Tiltfile` editing experience in IDEs/editors.
+
+Specifically, for the scope of this spec, we'll consider two high-level aspects:
+ * Syntax highlighting
+ * Advanced IDE features (autocomplete, go to definition, etc.)
+
+Explicitly out of scope for consideration here is any kind of integration with `tilt up` like was done for the [vscode-tilt-status][] extension.
+The focus here is on language-level (Starlark) static analysis.
```

Comment:
FWIW, I feel like vscode-tilt-status was really more of a tilt api demo, and a first pass at this feature should probably just open the Tilt web UI in a VSCode tab, which seems like pretty good bang:buck.